src/main/java/nextstep/subway/section/Sections.java

```diff
@@ -0,0 +1,53 @@
+package nextstep.subway.section;
+
+import nextstep.subway.station.domain.Station;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+@Embeddable
+public class Sections {
+
+    @OneToMany(mappedBy = "line", cascade = CascadeType.ALL)
+    List<Section> sections = new ArrayList<>();
+
+    public Sections() {
+    }
+
+    public void add(Section section) {
+        sections.add(section);
+    }
+
+    public boolean contains(Section section) {
+        return sections.contains(section);
+    }
+
+    public List<Section> getAllSection() {
+        return sections;
+    }
+
+    public List<Station> sortedStations() {
+        LinkedList<Station> stations = new LinkedList<>();
+
+        for (Section section : sections) {
+            addStation(stations, section);
+        }
+        return stations;
+    }
+
+    private LinkedList<Station> addStation(LinkedList<Station> stations, Section section) {
```

Comment:
```suggestion
    private List<Station> addStation(List<Station> stations, Section section) {
```
변경 가능하도록 인터페이스를 사용해주세요